<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QChaotiJulei</class>
 <widget class="QSubDialogBase" name="QChaotiJulei">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>超体聚类</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>空间权重（double）</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>种子分辨率（double）</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>体素分辨率（double）</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>应用</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://blog.csdn.net/wi162yyxq/article/details/72808643?locationNum=5&amp;amp;fps=1&quot;&gt;&lt;span style=&quot; font-size:18pt; text-decoration: underline; color:#aa007f;&quot;&gt;点击查看参考资料&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>颜色权重（double）</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit_5"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>向量权重（double）</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QSubDialogBase</class>
   <extends>QWidget</extends>
   <header>qsubdialogbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QChaotiJulei</receiver>
   <slot>OnApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnApply()</slot>
 </slots>
</ui>
